yoti-sdk-api/src/main/java/com/yoti/api/client/HumanProfile.java

```diff
@@ -44,6 +44,13 @@
      */
     Date getDateOfBirth();
 
+    /**
+     * Did the user pass the age verification check?
+     *
+     * @return <code>TRUE</code> if they passed, <code>FALSE</code> if they failed, <code>null</code> if there was no check
+     */
+    Boolean isAgeVerified();
+
```

Comment:
only one question, we need to provide also the implemented one in the HumanProfileAdapter, right?